src/main/resources/org/jenkinsci/tracey/RabbitMQTrigger/help-regexToEnv.html

```diff
@@ -4,5 +4,6 @@
         <li>Each line represents an extraction</li>
         <li>Each line must conform to the rule of KEY REGEX. There must be a space between the extraction key, and the regex. I.e <em>MYVALUE car=(\S+)</em></li>
         <li>We capture the value of the <em>FIRST</em> capture group in the regex.</li>
+        <li>I.e. BRANCH_FROM_MESSAGE \"branch":\\s+\"([^"']+)\"</li>
```

Comment:
You have to show a piece of the message to show from where you are about to extract this info.
Also, mention that two env variables will be injected
BRANCH_FROM_MESSAGE - will contain the whole match, i.e. branch = master
BRANCH_FROM_MESSAGE_1 - will contain first match group, i.e. master